spec/advanced_search_spec.rb

```diff
@@ -131,15 +131,15 @@
       end
       it 'subject as a phrase' do
         expect(@sub_phrase.size).to be >= 500
-        expect(@sub_phrase.size).to be <= 605
+        expect(@sub_phrase.size).to be <= 615
         expect(@sub_phrase).to have_fewer_results_than(solr_resp_doc_ids_only(subject_search_args("home schooling")))
         expect(@sub_phrase).to have_fewer_results_than @sub_no_phrase
       end
-      it 'keyword' do
+      it 'keyword', fixme: true do
         resp = solr_resp_doc_ids_only({ 'q' => 'Socialization' }.merge(solr_args))
-        # resp.should have_the_same_number_of_results_as(solr_resp_ids_from_query('Socialization'))
-        expect(resp.size).to be >= 475_000
-        expect(resp.size).to be <= 525_000
+        expect(resp).to have_the_same_number_of_results_as(solr_resp_ids_from_query('Socialization'))
```

Comment:
if you leave this line commented out, does it work?

Reply:
Yes, test will pass with line 140 commented out.